packages/account/src/Containers/Account/account.tsx

```diff
@@ -31,16 +31,30 @@ const Account = observer(({ history, location, routes }: TAccountProps) => {
         should_allow_authentication,
         should_allow_poinc_authentication,
     } = client;
-    const { toggleAccountSettings, is_account_settings_visible } = ui;
+    const { toggleAccountSettings, is_account_settings_visible, is_mobile, is_desktop } = ui;
+    const { is_passkeys_enabled } = React.useContext(PlatformContext);
+    const [available_routes, setAvailableRoutes] = React.useState(routes);
+    const { is_passkey_supported, is_loading } = useIsPasskeySupported();
+
+    const should_remove_passkey_route = !is_passkeys_enabled || is_desktop || (is_mobile && !is_passkey_supported);
+
+    React.useEffect(() => {
+        if (is_loading) return;
```

Comment:
why are we checking is_loading in useeffect?

Reply:
there is no sense to make further operations as should_remove_passkey_route is not fully calculated.
Because initially is_passkey_supported is false, but if device supports passkeys it will be changed